README.md

```diff
@@ -52,17 +67,32 @@ This command would need customizing for your environment.  [This link](https://s
 
 Once you have a certificate and key file name them `stash.crt` and `stash.key` and place them in the `~/.stash` directory.  Stash detects these and starts up using HTTPS rather than HTTP.
 
+# Customization
+
+You can make Stash interface fit your desired style with  [Custom CSS snippets](https://github.com/stashapp/stash/wiki/Custom-CSS-snippets) and [CSS Tweaks](https://github.com/stashapp/stash/wiki/CSS-Tweaks).
+
+-[Plex](https://github.com/stashapp/stash/wiki/Stash-Plex-Theme) is a community created theme inspired by popular Plex Interface.
+[Pulsar](https://github.com/stashapp/stash/wiki/Pulsar-Theme) is another option.
+
+
 # FAQ
 
 > I'm unable to run the app on OSX or Linux
 
 Try running `chmod u+x stash-osx` or `chmod u+x stash-linux` to make the file executable.
 
+> I cannot connect to the server from elsewhere within my network
+
+Changing the address of the server from 127.0.0.1 to the IP address of the computer on which you are running Stash
```

Comment:
`0.0.0.0` should be at least mentioned (and probably preferred). `Changing` should be `Change`. Add `.` to end of sentence.